org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/ContentModelCodeLensParticipant.java

```diff
@@ -91,6 +93,14 @@ public void doCodeLens(ICodeLensRequest request, List<CodeLens> lenses, CancelCh
 				createAssociateLens(documentURI, "Bind with xml-model", GrammarBindingType.XML_MODEL.getName(), range));
 	}
 
+	private static boolean canSupport(ICodeLensRequest request) {
+		if (!request.isSupportedByClient(CodeLensKind.Association)) {
+			return false;
+		}
+		String uri = request.getDocument().getDocumentURI();
+		return !DOMUtils.isXSD(uri) && !DOMUtils.isXSD(uri);

```

Comment:
`!DOMUtils.isXSD(uri)` --> `!DOMUtils.isDTD(uri)`

Reply:
or just remove the second occurence of `!DOMUtils.isXSD(uri)`, since it looks like Bind With CodeLens don't appear on DTD files as is.

Reply:
Fixed by testing isDTD